sys/net/gnrc/routing/rpl/gnrc_rpl_control_messages.c

```diff
@@ -546,7 +571,15 @@ void gnrc_rpl_recv_DIO(gnrc_rpl_dio_t *dio, kernel_pid_t iface, ipv6_addr_t *src
 #endif
         }
 
-        /* TODO: create prefix list entry */
```

Comment:
I don't know what I was thinking with this TODO comment. The code in master basically does, what I do below as well.